### PR TITLE
Feature/fix new round api key error 176

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: backend/flindt/integrations/messenger.py [hunk 1]
```diff
@@ -95,4 +98,5 @@ def send_message(self, message):
         # When in DEBUG mode also log the message to the console.
         if settings.DEBUG:
             logger.info(message)
-        self.provider.send_message(message)
+        if self.provider is not None:
```
Comment (ferdynice) [inline backend/flindt/integrations/messenger.py]: `if self.provider:` is sufficient and more clean

File: backend/flindt/feedback/models.py [hunk 2]
```diff
@@ -166,7 +166,7 @@ def save(self, *args, **kwargs):
 
         def send_feedback_received_message():
             message = _(
-                'You just received feedback. Read and rate it! https://{}/#/received-feedback/{}'.
+                'You just received feedback. Read and rate it! https://{}/received-feedback/{}'.
```
Comment (ferdynice) [inline backend/flindt/feedback/models.py]: Would normally put this in a separate ticket.

File: backend/flindt/feedback/models.py [hunk 3]
```diff
@@ -175,7 +175,7 @@ def send_feedback_received_message():
 
         def send_feedback_requested_received_message():
             message = _(
-                'The feedback you requested is now in Flindt. Read and rate it! https://{}/#/received-feedback/{}'.
+                'The feedback you requested is now in Flindt. Read and rate it! https://{}/received-feedback/{}'.
```
Comment (ferdynice) [inline backend/flindt/feedback/models.py]: Would normally put this in a separate ticket.

File: backend/flindt/integrations/messenger.py [hunk 4]
```diff
@@ -86,7 +86,10 @@ class Messenger(object):
     def __init__(self, *args, **kwargs):
         self.user = kwargs['user']
 
-        if self.user.slack_user_name and self.user.organization_set.first().slack_bot_api_key:
+        organization = self.user.organization_set.first()
+        if organization is None:
```
Comment (ferdynice) [inline backend/flindt/integrations/messenger.py]: `if not organization:` would be cleaner

File: backend/flindt/round/models.py [hunk 5]
```diff
@@ -64,9 +64,13 @@ def message_for_open(self):
         # Messenger expects a User object so return a list of Users for all selected users in this round.
         senders = User.objects.filter(id__in=sending_user_pks)
 
+        users_without_organization = []
         for sender in senders:
+            if not sender.organization_set.first():  # Check if all users (senders) have an organization
+                users_without_organization.append(sender)
             messenger = Messenger(user=sender)
             messenger.send_message(message)
+        return users_without_organization  # A list of users without organization, to be used in an admin message
```
Comment (ferdynice) [inline backend/flindt/round/models.py:75]: I would add this new behaviour to the docstring

File: backend/flindt/round/admin.py [hunk 6]
```diff
@@ -12,14 +12,17 @@ def start_round(self, request, obj):
...
-            obj.message_for_open()
+            users_without_organization = obj.message_for_open()
         except NoSolutionPossible:
             self.message_user(request,
                               'Not possible to start a round when users are included that do not fulfill any role!')
         except IntegrationError as e:
             self.message_user(request, 'There was an integration error: {}'.format(e))
         else:
             self.message_user(request, 'Round started!')
+            if len(users_without_organization):
```
Comment (ferdynice) [inline backend/flindt/round/admin.py]: `if users_without_organization:` is sufficient

File: backend/flindt/feedback/models.py (hunk 2 above)
Comment (ghost) [reply backend/flindt/feedback/models.py]: Yeah, I thought it was (and merged already) but I must've confused two branches before. #oops

**Pull request merged**